web/src/lib/components/TableFooter.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+    import type { Pagination } from '$lib/pagination';
+
+    export let pagination: Pagination;
+</script>
+
+{#if pagination.pagesCount > 1}
+<div
+    class="flex items-center justify-between border-t border-slate-200 bg-white px-4 py-3.5 sm:px-6"
+>
+    <nav class="flex flex-1 justify-end md:hidden" data-sveltekit-noscroll>
+        <a
+            href="{pagination.prevPageUrl}"
+            class="relative inline-flex items-center rounded-md border border-slate-300 bg-white px-4 py-2 text-sm font-medium text-slate-700"
+            class:hover:bg-slate-50={pagination.prevPageValid}
+            class:cursor-not-allowed={!pagination.prevPageValid}>Previous</a
+        >
+        <a
+            href="{pagination.nextPageUrl}"
+            class="relative ml-3 inline-flex items-center rounded-md border border-slate-300 bg-white px-4 py-2 text-sm font-medium text-slate-700"
+            class:hover:bg-slate-50={pagination.nextPageValid}
+            class:cursor-not-allowed={!pagination.nextPageValid}>Next</a
+        >
+    </nav>
+    <div class="hidden sm:flex-1 sm:items-center sm:justify-between md:flex">
+        <div>
+            <p class="text-sm text-slate-700">
+                Showing
+                <span class="font-semibold">{pagination.resultShowingFirst}</span>
+                to
+                <span class="font-semibold">{pagination.resultShowingLast}</span>
+                of
+                <span class="font-semibold">{pagination.resultsCount}</span>
+                results
+            </p>
+        </div>
+        <div>
+            <nav
+                class="isolate inline-flex -space-x-px rounded-md shadow-sm"
+                aria-label="Pagination"
+                data-sveltekit-noscroll
+            >
+                <a
+                    href="{pagination.prevPageUrl}"
```

Comment:
I think this should suffice
```suggestion
                    href={pagination.prevPageUrl}
```

Reply:
Or is this because of the potential `null` value?